test/integration/suite_test.go

```diff
@@ -33,11 +33,7 @@ var k8sClient *kubernetes.Clientset
 // Testing Main
 // -----------------------------------------------------------------------------
 
-func TestMain(m *testing.M) {
-	ctx, cancel = context.WithCancel(context.Background())
-	defer cancel()
-
-	fmt.Println("INFO: setting up test environment")
+func generateKongBuilder() (*kong.Builder, []string) {
```

Comment:
Without a named return variable it would be ideal to add a comment what this function returns.